### DISCUSSION TITLE
PWA Reload Propmt like the one in the docs

### DISCUSSION THREAD

**Discussion by SymphonyCR** [quasarframework/quasar#16882] [General - Components / Directives / etc]
Hi!, I'm wondering if anyone can share the component used in the docs to prompt the users to reload when a new version of the PWA app is available, right now I'm using the automatic reload, but it doesn't always work as expected and I prefer to let the users know that there is a new version available. 

If anyone could share the component and all what involves, I'll be so grateful, thanks in advance!

**Comment by kenyon-grublr** [comment]
They just use the Quasar Notify plugin. You can check out the service worker implementation in the repo found here:

https://github.com/quasarframework/quasar/blob/dev/docs/src-pwa/register-service-worker.js
